268:   keyboard_data.xkb_context = xkb_context_new(XKB_CONTEXT_NO_FLAGS);
--- a/src/vulkan/wsi/wayland.c
+++ b/src/vulkan/wsi/wayland.c
@@ -265,7 +265,7 @@
       abort();
    }
 
-   keyboard_data.xkb_context = xkb_context_new(XKB_CONTEXT_NO_FLAGS);
+   keyboard_data.xkb_context = xkb_context_new(XKB_CONTEXT_NO_DEFAULT_INCLUDES | XKB_CONTEXT_NO_ENVIRONMENT_NAMES);
 }
 
 static void
